src/rp2/plugin/report/abstract_ods_generator.py

```diff
@@ -172,9 +181,12 @@ def _fill_cell(
         style_name: str = f"{visual_style}_{data_style}"
         if isinstance(value, RP2Decimal):
             # The ezodf API doesn't accept RP2Decimal, so we are forced to cast to float before writing to the spreadsheet
-            value = float(value)
+            value = float(value) if value else 0.0
```

Comment:
Why the conditional? In this branch value is RP2Decimal, so conversion to float should always work.